### DISCUSSION TITLE
Update request values from Flow

### DISCUSSION THREAD

**Discussion by daniboy000** [postmanlabs/postman-flows#395] [Q&A]
Hello everyone. I did a simple example and what I want to do is to be able to change a request input value from Flow. In my example I will request weather data passing _Latitude_ as an input parameter, this value was initially defines as a _Number_ component, then I pass this value to a _Durable_ and define the name as **latitude**. Finally, this durable is connected to my request and a variable _latitude_ is created inside the request, reading the durable data. I though that creating the flow in that way I would be able to define a latitude using the number component, but this is not happening. Besides the changing in the latitude value, the request will always use the value that is defined in the original request. I would like to know what am I doing wrong or if it not possible to create a Flow to work that way. Follow bellow prints of my flow and the components configuration.

Flow
![flow](https://user-images.githubusercontent.com/1649166/190461921-53f83573-cd9b-49b2-96f1-6a5001d9681a.jpeg)

Latitude Durable
![latitude_durable](https://user-images.githubusercontent.com/1649166/190461935-842abcfe-64ac-43b1-a2fa-419721f03a96.jpeg)

Request
![request](https://user-images.githubusercontent.com/1649166/190461946-518ba081-cb7c-4b02-85e6-9c3bebe8da96.jpeg)

**Comment by samay-v** [comment]
@daniboy000  Can you try to do it the way this example solves this usecase? https://www.postman.com/postman/workspace/example-flows/flow/62fdd3360a222e16840ce44b
